tests/test_modeling_tf_encoder_decoder.py

```diff
@@ -0,0 +1,671 @@
+            EXPECTED_SUMMARY_STUDENTS = """sae was founded in 1856, five years before the civil war. the fraternity has had to work hard to change recently. the university of oklahoma president says the university's affiliation with the fraternity is permanently done. the sae has had a string of members in recent months."""
+
+            input_dict = tokenizer(ARTICLE_STUDENTS, return_tensors="tf")
+            output_ids = model.generate(input_ids=input_dict["input_ids"], max_length=None).numpy().tolist()
+            summary = tokenizer.batch_decode(output_ids, skip_special_tokens=True)
+
+            self.assertEqual(summary, [EXPECTED_SUMMARY_STUDENTS])
+
+
+@require_torch
```

Comment:
do we need torch here? Think just tensorflow is good enough no?

Reply:
The reason that we need PyTorch here is - currently, there is no TF weights on model hub, and in order to get the complete the same weights for testing, I need to load to PyTorch model first, then save to TF encoder/decoder. Then load them to `TFEncoderDecoderModel`. Once this PR is approved, we can upload the TF weights to Hub, and we can change this test (removing the torch dependency)

```
            # workaround to load from pt
            _model = EncoderDecoderModel.from_pretrained("patrickvonplaten/bert2bert-cnn_dailymail-fp16")
            _model.encoder.save_pretrained("./encoder")
            _model.decoder.save_pretrained("./decoder")
            model = TFEncoderDecoderModel.from_encoder_decoder_pretrained(
                "./encoder", "./decoder", encoder_from_pt=True, decoder_from_pt=True
            )
```

Reply:
my mistake :)